api/src/main/java/io/strimzi/api/kafka/model/AclRule.java

```diff
@@ -45,6 +49,14 @@ public AclRule(AclRuleType type, AclRuleResource resource, String host, AclOpera
         this.operation = operation;
     }
 
+    public AclRule(AclRuleType type, AclRuleResource resource, String host, List<AclOperation> operations) {
+        this.type = type;
+        this.resource = resource;
+        this.host = host;
+        this.operations = operations;
+    }
+
+
```

Comment:
Why do we need both this and the previous constructor without the `List<AclOperation> operations` field?

Reply:
It doesn't seem to be used anymore, indeed 👍